### PR TITLE
Add rest consumers stats

### PR DESCRIPTION
**Pull request by lwyic**
I'm adding "show message-vpn <vpnFilter> rest rest-consumer <itemFilter> stats" to the exporter.

### CODE REVIEW

**Review by GreenRover — APPROVED** [review]

File: semp/metricDesc.go [hunk 1]
```diff
@@ -477,4 +478,15 @@ var MetricDesc = map[string]Descriptions{
...
+		"http_post_response_messages_received":              NewSemDesc("rdp_http_post_response_messages_received", NoSempV2Ready, "The numer of response messages received.", variableLabelsRestConsumer),
+		"http_post_response_messages_received_successful":   NewSemDesc("rdp_http_post_response_messages_received_successful", NoSempV2Ready, "The numer of response messages received successfully.", variableLabelsRestConsumer),
+		"http_post_response_messages_received_error":        NewSemDesc("rdp_http_post_response_messages_received_error", NoSempV2Ready, "The numer of response messages received with errors", variableLabelsRestConsumer),
+		"http_post_request_bytes_sent":                      NewSemDesc("rdp_http_post_request_bytes_sent", NoSempV2Ready, "The request bytes sent.", variableLabelsRestConsumer),
+		"http_post_response_bytes_received":                 NewSemDesc("rdp_http_post_response_bytes_received", NoSempV2Ready, "The response bytes received.", variableLabelsRestConsumer),
```
Comment (GreenRover) [inline semp/metricDesc.go:490]: rename to `rdp_http_post_response_received_bytes`  to be prometheus complient.

File: semp/metricDesc.go [hunk 2]
```diff
@@ -477,4 +478,15 @@ var MetricDesc = map[string]Descriptions{
...
+		"http_post_request_messages_sent_timed_out":         NewSemDesc("rdp_http_post_request_messages_sent_timed_out", NoSempV2Ready, "The numer of time-out encounted while sending a message.", variableLabelsRestConsumer),
+		"http_post_response_messages_received":              NewSemDesc("rdp_http_post_response_messages_received", NoSempV2Ready, "The numer of response messages received.", variableLabelsRestConsumer),
+		"http_post_response_messages_received_successful":   NewSemDesc("rdp_http_post_response_messages_received_successful", NoSempV2Ready, "The numer of response messages received successfully.", variableLabelsRestConsumer),
+		"http_post_response_messages_received_error":        NewSemDesc("rdp_http_post_response_messages_received_error", NoSempV2Ready, "The numer of response messages received with errors", variableLabelsRestConsumer),
+		"http_post_request_bytes_sent":                      NewSemDesc("rdp_http_post_request_bytes_sent", NoSempV2Ready, "The request bytes sent.", variableLabelsRestConsumer),
```
Comment (GreenRover) [inline semp/metricDesc.go:489]: rename to `rdp_http_post_request_sent_bytes`  to be prometheus complient.

File: semp/metricDesc.go [hunk 3]
```diff
@@ -477,4 +478,15 @@ var MetricDesc = map[string]Descriptions{
 		"connection_fast_retransmit":      NewSemDesc("connection_fast_retransmit", NoSempV2Ready, "The number of TCP segments retransmitted due to the receipt of duplicate acknowledgments (‘ACKs’). See RFC 5681 for further details.", variableLabelsVpnClient),
 		"connection_timed_retransmit":     NewSemDesc("connection_timed_retransmit", NoSempV2Ready, "The number of TCP segments re-transmitted due to timeout awaiting an ACK. See RFC 793 for further details.", variableLabelsVpnClient),
 	},
+	"RestConsumerStats": {
+		"http_post_request_messages_sent":                   NewSemDesc("rdp_http_post_request_messages_sent", NoSempV2Ready, "The numer of request messages sent.", variableLabelsRestConsumer),
```
Comment (GreenRover) [inline semp/metricDesc.go:482]: Number of what kind of requests send to where?
Please be more specific, to make help text helpfull for non solace broker seniors.

**Review by GreenRover — CHANGES_REQUESTED** [review]

File: semp/metricDesc.go (hunk 1 above)
Comment (lwyic) [reply semp/metricDesc.go]: I've changed the names of the metrics to be more Prometheus compliant. Please check if it is ok. I've push the changes to my branch already. Do I need to create another pull request? Sorry, I'm new here.

File: semp/metricDesc.go (hunk 2 above)
Comment (lwyic) [reply semp/metricDesc.go]: I've changed the names of the metrics to be more Prometheus compliant. Please check if it is ok. I've push the changes to my branch already. Do I need to create another pull request? Sorry, I'm new here.

File: semp/metricDesc.go (hunk 3 above)
Comment (lwyic) [reply semp/metricDesc.go]: I've make the description more clear. Please check if it is ok. thanks.

**Review by GreenRover — APPROVED** [review]

**Comment by GreenRover** [comment]
Thank you for contributing.
I case you need a release i created:
https://github.com/solacecommunity/solace-prometheus-exporter/releases/tag/v1.8.2

**Pull request merged**